pages/index.js

```diff
@@ -134,6 +161,12 @@ export default function HomePage() {
 }
 
 const StyledGrid = styled.div`
-display: grid;
-grid-template-rows: repeat(3, auto);
-`
+  display: grid;
+  grid-template-rows: repeat(3, auto);
+`;
+
+const StyledTest = styled.section`
```

Comment:
Do you need this Test? If yes, I would recommend to rename it.

Reply:
Done, thanks!